From 82d127dd58231d46254b8344ac62f11c10a0c16c Mon Sep 17 00:00:00 2001
From: Jeff Daily <jeff.daily@amd.com>
Date: Thu, 26 Oct 2023 15:32:33 +0000
Subject: [PATCH] [ROCm] remove HCC references

rename __HIP_PLATFORM_HCC__ to WITH_ROCM

This symbol has had a long deprecation cycle and will finally be
removed in ROCm 6.0. The WITH_ROCM preprocessor symbol was already
defined by setup.py, so choosing to use it here instead to indicate the
same thing.
---
 torchvision/csrc/ops/cuda/deform_conv2d_kernel.cu | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/torchvision/csrc/ops/cuda/deform_conv2d_kernel.cu b/torchvision/csrc/ops/cuda/deform_conv2d_kernel.cu
index 0e82c5dd638..b4a44b03af9 100644
--- a/torchvision/csrc/ops/cuda/deform_conv2d_kernel.cu
+++ b/torchvision/csrc/ops/cuda/deform_conv2d_kernel.cu
@@ -82,7 +82,7 @@ namespace {
 const int kMaxParallelImgs = 32;
 
 inline unsigned int GET_THREADS() {
-#ifdef __HIP_PLATFORM_HCC__
+#ifdef WITH_HIP
   return 256;
 #endif
   return 512;
